src/features/workflow/RunBar.tsx

```diff
@@ -32,6 +32,8 @@ const RunBar: React.FC<RunBarProps> = ({
 
   const handleCancel = () => { alert('cannot cancel runs yet') }
 
+  const isMac = navigator.platform.indexOf('Mac') > -1
```

Comment:
move this to a new file `utils/platform.ts` & wrap it in an exported function:

```ts
export function platform(): 'mac' | 'pc' {
  if (navigator.playform.indexOf('Mac') > -1) {
    return 'mac'
  }
  return 'pc'
}
```

replace this line with: `const isMac = (platform() === 'mac')`